File: frontend/StudentManagement.Shared/Dtos/ThamSo/ThamSoResponse.cs
namespace StudentManagement.Shared.Dtos.ThamSo;

public class ThamSoResponse
{
    public string IdThamSo { get; set; } = string.Empty;
    public string TenThamSo { get; set; } = string.Empty;
    public int KieuThamSo { get; set; }
    public string GiaTriThamSo { get; set; } = string.Empty;
}

File: backend/src/main/java/com/demo/student_management/dto/thamso/ThamSoResponse.java
package com.demo.student_management.dto.thamso;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ThamSoResponse {
    private String idThamSo;
    private String tenThamSo;
    private Integer kieuThamSo;
    private String giaTriThamSo;
}
